internal/svcctl/svcctl.go

```diff
@@ -212,7 +215,10 @@ func waitUp(ctx context.Context, ipComm IPCommunicator, debugInfo *debugData) er
 }
 
 func stopAndWait(ctx context.Context, ipComm IPCommunicator) error {
-	debugInfo := newDebugData(ipComm, stopSvc, "")
+	debugInfo, err := newDebugData(ipComm, stopSvc, "")
+	if err != nil {
+		return locale.WrapError(err, "svcctl_cannot_create_debug_info", "Cannot create debug info: {{.V0}}", err.Error())
```

Comment:
I also realized this is a duplicate error message, so please add this to `en-us.yaml` and update the call on this line and line 159 with an empty string.